src/components/o-testimonials/_preview.html

```diff
@@ -0,0 +1,34 @@
+<axa-testimonials classes="o-testimonials"
+                  title="Customer Reviews"
+                  subtitle="AXA works hard to provide the best service possible to its customers.">
+  <div class="o-testimonials__item js-o-testimonial__item">
```

Comment:
@nakovskijosif 
@LucaMele 
@Ben-Zahler 
IMHO I don't like to have this repeating markup as children.
I could be generated by the template of the component and the data could be added through an `items="[{review, author}]"` attribute containing JSON payload.

If SEO is important here, it's okay. Though I don't have objective facts about SEO impacts by web-components 🤔

Reply:
@AndyOGo Yes we agreed that if is SEO relevant we give up a bit for the moment to be DRY.. I really think here is SEO relevant therefore ok for me.